pkg/apiserver/api_installer.go

```diff
@@ -128,6 +129,25 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		return err
 	}
 
+	// subresources must have parent resources, and follow the namespacing rules of their parent
+	if hasSubresource {
+		storage, ok := a.group.Storage[resource]
+		if !ok {
+			return fmt.Errorf("subresources can only be declared when the parent is also registered: %s needs %s", path, resource)
+		}
+		object := storage.New()
+		_, kind, err := a.group.Typer.ObjectVersionAndKind(object)
+		if err != nil {
+			return err
+		}
+		parentMapping, err := a.group.Mapper.RESTMapping(kind, a.group.Version)
+		if err != nil {
+			return err
+		}
+		mapping.Scope = parentMapping.Scope
```

Comment:
No - the scope of the child is immaterial in a sub resource.  Sub resources detach scope from kind - the scope is "the scope of my parent" always.